zendesk/ticket_test.go

```diff
@@ -163,3 +164,32 @@ func TestCreateTicket(t *testing.T) {
 		t.Fatalf("Returned ticket does not have the expected ID %d. Ticket id is %d", expectedID, ticket.ID)
 	}
 }
+
+func TestUpdateTicket(t *testing.T) {
+	mockAPI := newMockAPIWithStatus(http.MethodPut, "ticket.json", http.StatusOK)
+	client := newTestClient(mockAPI)
+	defer mockAPI.Close()
+
+	trg, err := client.UpdateTicket(ctx, 2, Ticket{})
+	if err != nil {
+		t.Fatalf("Failed to get ticket: %s", err)
+	}
+
+	expectedID := int64(2)
```

Comment:
are you trying to test that the url path was set correctly here or just that the client can handle the json response from the server?

Reply:
Removed tests serving no real function